src/app/tests.py

```diff
@@ -0,0 +1,33 @@
+import json
+
+from django.test import TestCase
+from django.urls import reverse
+from rest_framework.test import APIClient
+
+
+class TestAuthenticate(TestCase):
+    AUTHENTICATE_URL = reverse("authenticate")
+    FIRST_NAME = "Kate"
+    LAST_NAME = "Liang"
+    EMAIL = "ksl67@cornell.edu"
```

Comment:
you shouldn't be advertising your email -- create a test user and make that read in from a constants file somewhere